USING_ELEKTRA.md

```diff
@@ -0,0 +1,252 @@
+# Using Elektra
+
+In the new version of LCDproc, [Elektra](https://www.libelektra.org/home) 
+will be used to store and edit configuration settings which are used by LCDproc.
+
+Elektra stores configuration in a global, hierarchical key database. 
+Elektra furthermore provides the possibility to have a 
+[validation on configuration settings](https://www.libelektra.org/tutorials/validate-configuration)
```

Comment:
This reads like that validation will be the only difference, please add other reasons, too.

Reply:
You also wrote this sentence (one of you first comments on kodebachs review)

> I think we do not need any advertisement here. Simply tell what a user can do within this tutorial. Links within the tutorial where something is mentioned.

What exactly should I write here so that it is no advertisement?

Reply:
Yes, it is tricky. If you describe what the tutorial will show, you are on the safe side.